app/(golems)/golems/components/MermaidDiagram.tsx

```
@@ -38,30 +38,30 @@ export default function MermaidDiagram({ chart }: { chart: string }) {
+      <div className="mb-4 flex min-h-[100px] items-center justify-center rounded-lg border border-[#e5950026] bg-[#0d0d0d] p-4">
+        <span className="text-sm text-[#a89078]">Loading diagram...</span>
       </div>
     );
   }
 
   return (
     <div
       ref={containerRef}
-      className="bg-[#0d0d0d] border border-[#e5950026] rounded-lg p-4 mb-4 overflow-x-auto [&_svg]:max-w-full [&_svg]:mx-auto flex justify-center"
+      className="mb-4 flex justify-center overflow-x-auto rounded-lg border border-[#e5950026] bg-[#0d0d0d] p-4 [&_svg]:mx-auto [&_svg]:max-w-full"
       dangerouslySetInnerHTML={{ __html: svg }}
```

Biome (2.4.6)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)